src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public Ball(int number, Colors color) {
+        this.number = number;
+        this.color = String.valueOf(color);
+    }

```

Comment:
```suggestion
    public Ball(int number, String color) {
        this.number = number;
        this.color = color;
    }
```
please do casting from enum to String in a different place
constructor should not contain logic